### DISCUSSION TITLE
Activate camera to record video when receiver is picked up?

### DISCUSSION THREAD

**Discussion by xentelworker** [nickpourazima/rotary-phone-audio-guestbook#64] [Ideas]
Is there any way to activate a camera to record video as people are leaving audio messages?

**Comment by nickpourazima** [comment]
Yeah definitely! 

There would just need to be a GPIO trigger pulse to be sent out when a user lifts the phone off the hook.

Depending on the camera system you might need some components along the path but take a look at this to get some ideas:

https://cudmore.github.io/triggercamera/

**Comment by digitaleJugend** [comment]
That would be great to use the Pi-Cam for video recording. I saw some hungarian phone with a pi cam installed in phone and with the round Touch Display from Raspberry. To Use is also as Photobooth. During the Call you see also the Videopicture on the Display as reference. Also they have an Screensaver for the Event. https://www.memoringphone.com


![2025-01-19 12_49_50-Window](https://github.com/user-attachments/assets/eba48ba8-2b00-4cbb-a6cf-805e39e31c94)
